echopype/tests/mask/test_mask.py

```diff
@@ -232,21 +236,21 @@ def create_input_mask(
 @pytest.mark.parametrize(
     ("n", "n_chan_freq", "add_chan", "add_freq_nom", "freqAB", "chanAB"),
     [
-        (5, 3, True, True, [1.0, 3.0], None),
-        (5, 3, True, True, None, ['chan1', 'chan3']),
-        pytest.param(5, 3, False, True, [1.0, 3.0], None,
+        (5, 4, True, True, [1000.0, 2.0], None),
+        (5, 4, True, True, None, ['chan1', 'chan3']),
+        pytest.param(5, 4, False, True, [1.0, 2000000.0], None,
                      marks=pytest.mark.xfail(strict=True,
                                              reason="This should fail because the Dataset "
                                                     "will not have the channel coordinate.")),
-        pytest.param(5, 3, True, False, [1.0, 3.0], None,
+        pytest.param(5, 4, True, False, [1.0, 2.0], None,
                      marks=pytest.mark.xfail(strict=True,
                                              reason="This should fail because the Dataset "
                                                     "will not have the frequency_nominal variable.")),
-        pytest.param(5, 3, True, True, [1.0, 4.0], None,
+        pytest.param(5, 4, True, True, [1.0, 4.0], None,
                      marks=pytest.mark.xfail(strict=True,
                                              reason="This should fail because not all selected frequencies"
                                                     "are in the frequency_nominal variable.")),
-        pytest.param(5, 3, True, True, None, ['chan1', 'chan4'],
+        pytest.param(5, 4, True, True, None, ['chan1', 'chan9'],
```

Comment:
I'd suggest another 2 cases where you xfail duplicated `freqAB` and duplicated `chanAB` cases, just to be complete.